content/docs/editing/single-document-collections.mdx

```diff
@@ -38,6 +38,8 @@ export default defineConfig({
 
 Once both settings have been applied, and the document contains a single file, the UI will forego the list page and direct the user straight to the document.
 
+<WebmEmbed embedSrc="/video/Single Collection Demo.webm"/>
```

Comment:
**issue**: Avoid spaces in names - https://www.ssw.com.au/rules/remove-spaces-from-your-folders-and-filename/

Reply:
Resolved